full-service/src/service/wallet.rs

```diff
@@ -196,6 +225,11 @@ pub enum JsonCommandResponse {
     },
 }
 
+// Helper method to format displaydoc errors in json.
+fn format_error<T: std::fmt::Display + std::fmt::Debug>(e: T) -> String {
+    json!({"error": format!("{:?}", e), "details": e.to_string()}).to_string()
```

Comment:
Yay!

Reply:
I actually think error and details should be switched because the debug formatting will have more information.

Reply:
It's sort of counter-intuitive. The debug only prints the name of the error, and the to_string() prints the full displaydoc string.

Reply:
Ha, yeah, I can see that being the case. My mistake then :)